x-pack/solutions/security/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -390,103 +501,66 @@ export const SessionView = ({
           </EuiFlexItem>
 
           <EuiFlexItem grow={false}>
-            <EuiButton
-              onClick={toggleDetailPanel}
-              iconType="list"
-              data-test-subj="sessionView:sessionViewDetailPanelToggle"
-              fill={!isDetailOpen}
-            >
-              {DETAIL_PANEL}
-            </EuiButton>
+            {openDetailsInExpandableFlyout ? (
+              <EuiButtonIcon onClick={toggleDetailPanelInFlyout} iconType="expand" />
```

Comment:
Should we maybe remove the button on the preview flyout mode? since it can be closed by clicking on the close button on the flyout (what doesn't have on the details panel), and it can be expanded by clicking on the process.

I think the expand icon might lead users to think it's a fullscreen option, also it doesn't do anything when the preview is already expanded.

https://github.com/user-attachments/assets/ca92fc78-46cf-4ebb-a062-ce6d770ad9b6

Reply:
we can, but this was added to be consistent with the analyzer graph component, and was a request by the UIUX team. I'll check with Product and UIUX if they have a preference. I'll get back to you on it!